src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java

```diff
@@ -150,12 +153,12 @@ public boolean canCommandSenderUseCommand(Commands theCommand)
     @NotNull
     public boolean isPlayerOpped(@NotNull final ICommandSender sender, String cmdName)
     {
-        int requiredOpLevel = PERMNUM;
-        if (PERMNUM < 1)
+        int requiredOpLevel = Configurations.opLevelForServer;
+        if (Configurations.opLevelForServer < 1)
         {
             requiredOpLevel = 1;
         }
-        if (PERMNUM > 4)
+        if (Configurations.opLevelForServer > 4)
```

Comment:
I just remembered, does MathUtils.clamp(Value,1,4) not clamp the value between 1 and 4?